deploy.cfg.tmpl

```diff
@@ -69,7 +69,11 @@ schema-collection = {{ default .Env.schema_collection "samples_schema" }}
 
 # A URL pointing to a configuration file for any metadata validators to be installed on startup.
 # See the readme file for a description of the file contents.
-metadata-validator-config-url = {{ default .Env.metadata_validator_config_url "https://raw.githubusercontent.com/kbase/sample_service_validator_config/master/metadata_validation.yml" }}
+metadata-validator-config-repo = {{ default .Env.metadata_validator_config_repo "kbase/sample_service_validator_config" }}
+metadata-validator-config-filename = {{ default .Env.metadata_validator_config_filename "metadata_validation.yml" }}
+metadata-validator-config-prerelease = {{ default .Env.metadata_validator_config_repo "0" }}
+metadata-validator-config-url = {{ default .Env.metadata_validator_config_url "" }}
+github-token = {{ default .Env.github_token "" }}
```

Comment:
do we want this to default to an empty string, or to fail? The token is required, with no sensible default.
same for the config url.
so, rather,
`{{ .Env.github_token }}`
etc.

Also, this change will require synchronized changes to wherever the deployment configs are kept - probably gitlab.

Reply:
a good point